README.md

```diff
@@ -1,55 +1,385 @@
-# `Saturn` — parallelism-safe data structures for Multicore OCaml
+[API reference](https://ocaml-multicore.github.io/saturn/)
 
----
+<!--
+```ocaml
+# #thread
+```
+-->
+
+# Saturn — parallelism-safe data structures for Multicore OCaml
 
 This repository is a collection of parallelism-safe data structures for OCaml 5.
 They are contained in two packages:
 
-- `Saturn` that includes every data structures and should be used by default if
-  you just want parallelism-safe data structures..
-- `Saturn_lockfree` that includes only lock-free data structures.
+- Saturn that includes all data structures and should be used by default if you
```

Comment:
Maybe it's useful to make it clear that Saturn exposes both lockfree and other parallelism-safe data structures.